lib/IRGen/GenDistributed.cpp

```diff
@@ -486,6 +486,7 @@ void DistributedAccessor::decodeArgument(unsigned argumentIdx,
     // The argument is +0, so we can use the address of the param in
     // the context directly.
     arguments.add(resultAddr);
+    LoadedArguments.push_back(std::make_pair(resultValue.getAddress(), argumentType));
```

Comment:
Oh, I vaguely remember that it was decided unnecessary when we did the other PR…